src/renderer/components/proxy-settings/proxy-settings.js

```diff
@@ -124,27 +123,29 @@ export default Vue.extend({
       if (!this.useProxy) {
         this.enableProxy()
       }
-      $.getJSON(this.proxyTestUrl, (response) => {
-        console.log(response)
-        this.proxyIp = response.ip
-        this.proxyCountry = response.country
-        this.proxyRegion = response.region
-        this.proxyCity = response.city
-        this.dataAvailable = true
-      }).fail((xhr, textStatus, error) => {
-        console.log(xhr)
-        console.log(textStatus)
-        console.log(error)
-        this.showToast({
-          message: this.$t('Settings.Proxy Settings["Error getting network information. Is your proxy configured properly?"]')
+      fetch(this.proxyTestUrl)
+        .then((response) => response.json())
+        .then((json) => {
+          console.log(json)
```

Comment:
I know you added this just because it was in the previous code but i think we should remove this console.log

Reply:
Removed it now.